util/weight_gen/run.hercules.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+#SBATCH -J weight_gen
+#SBATCH -A fv3-cpu
+#SBATCH --open-mode=truncate
+#SBATCH -o log
+#SBATCH -e log
+#SBATCH --ntasks=1
+#SBATCH -q debug
+#SBATCH -t 00:03:00
+
+#-------------------------------------------------------------------------------
+#
+# Run the weight_gen program on Orion.
```

Comment:
I think you mean 'Hercules'.

This is my last comment on this PR. Will merge after this fix.

Reply:
Good catch!  Fixed.